test/run.sh

```diff
@@ -65,9 +91,8 @@ for dockerImage in "$@"; do
 		continue
 	fi
 	
-	noNamespace="${dockerImage##*/}"
```

Comment:
So, the theory here was that you could run tests on something like `myuser/mysql` and have it run the `mysql` tests appropriately -- I'm not sure that I've ever actually used that functionality myself, but I figure it's worth pointing out that this would be a change in case it matters to anyone (I certainly don't mind, since I've never actually used it).